docs/schema/index.md

```diff
@@ -4,7 +4,7 @@ The Open Contracting Data Standard is maintained using JSON Schema.
 
 In this section you will find the schemas for [releases](release) and [records](record) along with the schemas for [packaging](packaging/index.md), which act as envelopes for releases and records.
 
-The [release schema reference](reference) provides guidance on using each of the [sections](reference.md#release-structure) and [building blocks](reference.md#building-block-reference) in the schema, and the [record schema reference](records_reference) provides additional information on publishing records with compiled and versioned releases.
+The [release schema reference](reference) provides guidance on using each of the [sections](reference.md#release-structure) and [subschemas](reference.md#subschema-reference) in the schema, and the [record schema reference](records_reference) provides additional information on publishing records with compiled and versioned releases.
```

Comment:
Use field/object/array instead of section.

```suggestion
The [release schema reference](reference) provides guidance on using each of the [fields](reference.md#release-structure) and [subschemas](reference.md#subschema-reference) in the schema, and the [record schema reference](records_reference) provides additional information on publishing records with compiled and versioned releases.
```